x/ibc/05-port/keeper/keeper.go

```diff
@@ -16,6 +14,7 @@ type Keeper struct {
 	cdc       *codec.Codec
 	codespace sdk.CodespaceType
 	prefix    []byte // prefix bytes for accessing the store
+	ports     map[sdk.CapabilityKey]string
```

Comment:
NOTE: we still want to be able to query the existing ports

Reply:
Added `GetPorts`.